src/templates/finops-hub/schema/schema_ea_normalized.json

```
@@ -0,0 +1,723 @@
+          "name": "ExchangeRatePricingToBilling",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "Frequency",
+          "type": "String"
+        },
+        "sink": {
+          "name": "Frequency",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "invoiceId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "InvoiceId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "InvoiceSectionId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "InvoiceSectionId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "InvoiceSectionName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "InvoiceSectionName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "IsAzureCreditEligible",
+          "type": "String"
+        },
+        "sink": {
+          "name": "IsAzureCreditEligible",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "location",
+          "type": "String"
+        },
+        "sink": {
+          "name": "Location",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterCategory",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterCategory",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterRegion",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterRegion",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterSubCategory",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterSubCategory",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "paygCostInBillingCurrency",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PaygCostInBillingCurrency",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "paygCostInUsd",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PaygCostInUsd",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "PayGPrice",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PayGPrice",
```

Can we fix the mis-capitalized "G"?
```suggestion
          "name": "PaygPrice",
```

---

Done